core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -468,21 +484,50 @@ private void handleCacheStreamException(IOException e) {
   }
 
   /**
-   * Only updates {@link #mCurrentCacheStream}, {@link #mCurrentBlockInStream} and
-   * {@link #mStreamBlockId} to be in sync with the current block (i.e. {@link #getCurrentBlockId()}
-   * ). If this method is called multiple times, the subsequent invokes become no-op. Call this
-   * function every read and seek unless you are sure about the block streams are up-to-date.
+   * Checks whether the current block stream has remaining bytes, if not, advances to the next block
+   * stream. Otherwise, this is a no-op.
+   *
+   * This should only be used in streaming read APIs. For APIs like {@link #seek(long)},
+   * {@link #skip(long)}, the block stream should be updated according to the current read position,
+   * use {@link #updateStreamsBasedOnPosition()} instead.
    */
   private void updateStreams() throws IOException {
```

Comment:
I would call this method `updateStreamsOnRead` to make it more explicitly who should call this method

Reply:
nice, thanks.